nixos/lib/make-options-doc/mergeJSON.py

```diff
@@ -208,19 +236,48 @@ def optionIs(option: Dict[str, Any], key: str, typ: str) -> bool:
         if '_type' not in option[key]: return False
         return option[key]['_type'] == typ
 
+    def convertCode(name: str, option: Dict[str, Any], key: str):
+        if optionIs(option, key, 'literalMD'):
+            option[key] = convertString(name, f"*{key.capitalize()}:*\n{option[key]['text']}")
```

Comment:
We're breaking the implicit convention, currently relied upon by nixos-search, that options.json has the same format as what `lib.optionAttrSetToDocList` outputs (up to changing a list into a dict indexed by `name`), and making it closer to an internal data structure destined for `options-to-docbook.xsl`.

We should either undo that (IMO best for now), or revert https://github.com/NixOS/nixos-search/pull/460 first.

Reply:
> does nixos-search inspect relatedPackages as well?

Nope

Reply:
> once we get rid of options-to-docbook.xsl

What's the plan for that? Wouldn't we need to forbid DocBook descriptions / literalDocBook first?